runtime/ajuna/src/lib.rs

```diff
@@ -503,13 +501,8 @@ parameter_types! {
 
 impl pallet_treasury::Config for Runtime {
 	type Currency = Balances;
-	type ApproveOrigin = EnsureRootOrMoreThanHalfCouncil;
 	type RejectOrigin = EnsureRootOrMoreThanHalfCouncil;
 	type RuntimeEvent = RuntimeEvent;
-	type OnSlash = ();
-	type ProposalBond = FivePercent;
-	type ProposalBondMinimum = MinimumProposalBond;
-	type ProposalBondMaximum = MaximumProposalBond;
```

Comment:
All these values have already been deprecated (https://github.com/paritytech/polkadot-sdk/issues/138), since a long time ago. And have now simply been removed, https://github.com/paritytech/polkadot-sdk/pull/4831. So we can just drop those without replacement.